src/containers/Plot/index.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from "react";
+import { connect } from "react-redux";
+import PlotlyPlot from "react-plotly.js";
+// import { ActionCreator } from "redux";
+
+import { State } from "../../state/types";
+import { PlotConfig } from "../ResultsPanel/types";
+import { getCurrentTime } from "../../state/selection/selectors";
+// import { changeTime } from "../../state/selection/actions";
+// import { ChangeTimeAction } from "../../state/selection/types";
+import { PLOT_STYLE } from "../ResultsPanel/constants";
+
+interface PlotProps {
+    plotConfig: PlotConfig;
+    time: number;
+    // changeTime: ActionCreator<ChangeTimeAction>;
```

Comment:
remove commented out code (here and in the imports)?

Reply:
I left them in thinking we might bring these back soon, but I can just remove them too :)

Reply:
oh, because of updating the timebar, yeah, I might put a TODO somewhere that says "allow dragging of timebar to update app time" or something so it doesn't just look like forgotten code.